Daytime-Operations/Safety/Control-Safety-Systems.rst

```diff
@@ -27,6 +27,9 @@ Control Safety Systems
 Presented here is an overview of the concepts LOTO (Lockout/Tagout), GIS (Global Interlock System), and E-STOP (Emergency Stop), and when each procedure or system is used. 
 Specific procedures, like how to LOTO the Telescope Mount Assembly (TMA), or how to reset the GIS to resume operations of a specific component, are not described here. 
 The final procedures for LOTO will be documented on the Observatory Operations pages and linked here. 
+
+    - :ref:`Auxiliary-Telescope-AuxTel-LOTO-procedure`
```

Comment:
In my build I am not seeing a link here, just the text. I think it's missing the <Daytime-Operations-Auxiliary-Telescope-AuxTel-LOTO-procedure.html>`

Reply:
👍